README.md

```diff
@@ -19,16 +19,16 @@ Chrome has been shipping its own nonstandard segmentation API called `Intl.v8Bre
 let segmenter = new Intl.Segmenter("fr", {granularity: "word"});
 
 // Get an iterator over a string
-let iterator = segmenter.segment("Ceci n'est pas une pipe");
+let iterator = segmenter.break("Ceci n'est pas une pipe");
 
 // Iterate over it!
-for (let {segment, breakType} of iterator) {
-  console.log(`segment: ${segment} breakType: ${breakType}`);
+for (let {index, breakType} of iterator) {
+  console.log(`location: ${index} breakType: ${breakType}`);
   break;
 }
 
 // logs the following to the console:
-// segment: Ceci breakType: letter
+// location: 4 breakType: letter
```

Comment:
Index 4 only makes sense if we identify boundaries by their preceding index, which means the first boundary is at -1 rather than 0. Is that your intention?